src/instruments/src/EFB/Ground/Ground.tsx

```diff
@@ -17,47 +17,105 @@
  */
 
 import React from 'react';
-import { IconCornerDownLeft, IconCornerDownRight, IconArrowDown, IconHandStop, IconTruck, IconBriefcase, IconBuildingBridge, IconBuildingArch, IconArchive } from '@tabler/icons'
+import { IconCornerDownLeft, IconCornerDownRight, IconArrowDown, IconHandStop, IconTruck, IconBriefcase, IconActivity, IconBuildingArch, IconArchive } from '@tabler/icons'
 
 import './Ground.scss'
 import fuselage from '../Assets/320neo-outline-upright.svg'
+import {setSimVar, getSimVar} from '../../util.mjs'
 
 type GroundProps = {}
 
-type GroundState = {}
+type GroundState = {
+    tugStatus: boolean;
+}
 
 class Ground extends React.Component<GroundProps, GroundState> {
 
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            tugStatus: false,
+        };
+      }
+
+    toggleGroundAction(action: GroundServices) {
+        return (() =>{
+            setSimVar(action,"1","boolean");
```

Comment:
Please fix all spacing issues throughout the code.

```suggestion
            setSimVar(action, "1",  "boolean");
```